libs/cardano-ledger-core/testlib/Test/Cardano/Ledger/Common.hs

```diff
@@ -8,8 +8,12 @@ module Test.Cardano.Ledger.Common (
   -- * Expr
   ToExpr (..),
   showExpr,
+  ansiExpr,
+  ansiExprString,
   diffExpr,
+  diffExprString,
```

Comment:
Feel free to keep the names the way you've done them, if you'd like. But, I just wanted to make a comment for the future.

I would have added instead a new function, named something like `diffDocExpr`, while keeping old one `diffExpr` intact. That would have resulted in a lot less API breakage. (Same comment applies to changes of functions in this PR)

It is always good practice to keep breakage to a minimum. In this case it is just test helper functions, which probably don't have much impact downstream. But for regular ledger API this comment is very much applicable.